style.css

```diff
@@ -73,15 +77,220 @@ img {
+.social_media_about {
+  display: flex;
+  flex-direction: column;
+  padding: 10px 24px;
+}
+
+.social_media_about a {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
 }
 
 .social_icons {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style: none;
+  gap: 30px;
+}
+
+.social_icons li i {
+  width: 24px;
+  height: 24px;
+  color: #505f79;
+}
+
+#social_icons {
+  padding-left: 8px;
+}
+
+.social_media h3 {
+  margin: 0;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.works {
+  background: #e5e5e5;
+  display: grid;
+  grid-template-columns: auto;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+}
+
+.card {
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+}
+
+.project_name {
+  color: #172b4d;
+  margin-top: 12px;
+}
+
+#title {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.project_info {
+  display: flex;
+  gap: 5px;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.ul {
+  display: flex;
+}
+
+.company p {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  text-transform: uppercase;
+}
+
+.dot {
+  width: 8px;
+  height: 8px;
+}
+
+.role {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  margin-right: 30px;
+}
+
+.year {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.description {
+  width: 90%;
+  margin-top: 10px;
+}
+
+.description p {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.languages {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+  margin-top: 12px;
+  list-style: none;
+  padding: 0;
+}
+
+#lang {
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.button-container {
+  border: 1px solid #6070ff;
+  background: #fff;
+  color: #396df2;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  border-radius: 8px;
+  padding: 12px;
+  width: 162px;
+}
+
+.button-container:hover {
+  background: #6070ff;
+  color: #fff;
+  transition: all 0.5s ease-out;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 24px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  padding: 12px 24px 0 24px;
+}
+
+.skills {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.skills h2 {
+  color: #000;
+  list-style-type: none;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.skills img {
   display: flex;
   align-items: center;
+  width: 24px;
+  height: 24px;
+}
+
+.langList {
   list-style: none;
-  filter: grayscale(100%);
+  padding: 0;
+}
+
+.langImg {
+  width: 30vw;
+}
+
```

Comment:
- [ ]  Please fix the issue, you can try to delete the width:30vw.
![ScreenShot_20221029190249](https://user-images.githubusercontent.com/89518488/198841444-482256af-103b-468b-802b-ba4e9c89051a.png)